Power_Supervisor.lib

```diff
@@ -140,6 +140,50 @@ X VSS 3 -100 -400 100 U 50 50 1 1 W
 ENDDRAW
 ENDDEF
 #
+# MCP39F521
+#
+DEF MCP39F521 U 0 20 Y Y 1 F N
+F0 "U" -450 750 50 H V C CNN
+F1 "MCP39F521" 350 750 50 H V C CNN
+F2 "Package_DFN_QFN:QFN-28-1EP_5x5mm_P0.5mm_EP3.35x3.35mm" 0 100 50 H I C CNN
+F3 "" 0 100 50 H I C CNN
+$FPLIST
+ QFN*1EP*5x5mm*P0.5mm*EP3.35x3.35mm*
+$ENDFPLIST
+DRAW
+S -500 700 500 -800 0 1 10 f
+X EVENT 1 600 -100 100 L 50 50 1 1 O
+X ~RESET 10 -600 600 100 R 50 50 1 1 I
+X AVDD 11 -100 800 100 D 50 50 1 1 W
+X A0 12 600 400 100 L 50 50 1 1 I
+X SCL 13 600 200 100 L 50 50 1 1 I
+X SDA 14 600 100 100 L 50 50 1 1 B
+X A1 15 600 300 100 L 50 50 1 1 I
+X I1+ 16 -600 400 100 R 50 50 1 1 I
+X I1- 17 -600 300 100 R 50 50 1 1 I
+X V1- 18 -600 100 100 R 50 50 1 1 I
+X V1+ 19 -600 200 100 R 50 50 1 1 I
+X NC 2 -500 500 100 R 50 50 1 1 N N
+X AN_IN 20 -600 -400 100 R 50 50 1 1 I
+X AGND 21 -100 -900 100 U 50 50 1 1 W
+X ZCD 22 600 -200 100 L 50 50 1 1 O
+X REFIN+/OUT 23 600 -400 100 L 50 50 1 1 B
+X DGND 24 100 -900 100 U 50 50 1 1 W
+X DVDD 25 100 800 100 D 50 50 1 1 W
+X ~MCLR 26 600 600 100 L 50 50 1 1 I
+X DGND 27 100 -900 100 U 50 50 1 1 P N
+X ~DR 28 600 -600 100 L 50 50 1 1 O
+X DGND 29 100 -900 100 U 50 50 1 1 P N
+X NC 3 -500 0 100 R 50 50 1 1 N N
+X COMMON_B 4 -600 -200 100 R 50 50 1 1 I
+X COMMON_A 5 -600 -100 100 R 50 50 1 1 O
```

Comment:
Shouldn't `COMMONA` and `COMMONB` be passive? I couldn't find a reference to them being output and input respectively.